bcl/src/bc_module_sensor.c

```diff
@@ -179,3 +185,74 @@ void bc_module_sensor_toggle_output(bc_module_sensor_channel_t channel)
 {
     bc_gpio_toggle_output(_bc_module_sensor_channel_gpio_lut[channel]);
 }
+
+bool bc_module_sensor_set_vdd(bool on)
+{
+    uint8_t port_actual;
+    uint8_t port_new;
+
+    port_actual = _bc_module_sensor.tca9534a._output_port;
+
+    if (on)
+    {
+        port_new = port_actual & ~_BC_MODULE_SENSOR_VDD;
+    }
+    else
+    {
+        port_new = port_actual | _BC_MODULE_SENSOR_VDD;
+    }
+
+    if (port_actual != port_new)
+    {
+        return bc_tca9534a_write_port(&_bc_module_sensor.tca9534a, port_new);
+    }
+
+    return true;
+}
+
+bc_module_sensor_revision_t bc_module_sensor_get_revision(void)
```

Comment:
How does it work, is it using parasite capacitance of the pin?

Reply:
two pins of the expander are connected with 10k resistor https://github.com/bigclownlabs/bc-hardware/blob/master/out/bc-module-sensor/bc-module-sensor-rev-1-1-sch.pdf